terragraph_planner/los/test/test_ellipsoidal_los_validator.py

```diff
@@ -85,11 +87,30 @@ def test_out_of_distance_range(self) -> None:
             0.0,
         )
 
+    def test_large_el_dev(self) -> None:
```

Comment:
If you put the test_large_el_dev in test_base_los_validator.py, you can only keep one test case.